src/display.js

```diff
@@ -841,6 +841,22 @@ SpiceDisplayConn.prototype.hook_events = function()
     {
         var canvas = this.surfaces[this.primary_surface].canvas;
```

Comment:
`var`?

Reply:
Yes, I kept the same syntax as the whole file